### DISCUSSION TITLE
Comments Engine

### DISCUSSION THREAD

**Discussion by giscus** [hbstack/site#769] [General]
# Comments Engine

Comments Engine

https://hbstack.dev/modules/blog/comments/

**Comment by LITUATUI** [comment]
Please add support to [Emote](https://emote.com/) comment system.

Right now I'm testing it and inserted the HTML code into the hb-blog-post-comments.html hook.

HTML code:

```
<script defer src="https://i.emote.com/js/emote.js"></script>
<div id="emote_com"></div>
```

**Reply by razonyang** [reply]
Seems not a popular and complete service, I doubt that if it match the dark mode perfectly, at least I didn't find any docs so far.

**Reply by razonyang** [reply]
FWIW, here you go, [github.com/hbstack/blog/modules/emote/v0.1.0](https://github.com/hbstack/blog/releases/tag/modules/emote/v0.1.0).

**Reply by LITUATUI** [reply]
It's a new service, still beta. Thanks!

**Reply by FuadEfendi** [reply]
I think Giscus is best from SEO viewpoint: all comments are searchable by Google and point to your website page from high-ranking GitHub; plus, GitHub's "search" widget is usually shown in Google SERP. All comments are published on separate site as static HTML and it is super powerful SEO.

The only why want to check other solutions is that with GitHub users need to create account at GitHub... but if it is *real* user and not scammer, should not be an issue.

**Reply by FuadEfendi** [reply]
LOL ;) Emote admin UI is exactly the same as https://commento.io/, including installation script etc.; I am already evaluation Commento which is not free if cloud-hosted.
